src/box/sql/vdbe.c

```diff
@@ -466,7 +466,182 @@ int sqlVdbeExec(Vdbe *p)
 			printf("VDBE Trace:\n");
 	}
 #endif
-	for(pOp=&aOp[p->pc]; 1; pOp++) {
+
+#define DISPATCH() \
+do{ \
+pOp++; \
```

Comment:
Fix indentation (do block should be tabbed); place spaces before/after {; add a comment

Reply:
Done.